default/scripting/policies/METROPOLES.focs.txt

```diff
@@ -3,28 +3,36 @@ Policy
     description = "PLC_METROPOLES_DESC"
     short_description = "PLC_METROPOLES_SHORT_DESC"
     category = "ECONOMIC_CATEGORY"
-    adoptioncost = 1.0 * floor(5 + 0.25 * Statistic Sum value = LocalCandidate.Population
-                                          condition = And [ Planet OwnedBy empire = Source.Owner ])
+    adoptioncost = 1.0 * floor(15 + 0.25 * Statistic Sum value = LocalCandidate.Population
+                                           condition = And [ Planet OwnedBy empire = Source.Owner ])
     exclusions = [ "PLC_CONFEDERATION" "PLC_COLONIZATION" ]
     effectsgroups = [
         [[SPECIES_LIKES_OR_DISLIKES_POLICY_STABILITY_EFFECTS]]
 
-        // TODO: limit of planets based on number of regional admins?
-
-        // boosts to largest-population planets...
+        // Boost largest-population planets, provided they have enough population and stability.
+        // Metropoles need some "umland", so no more than 1 in 4 can become a metropole.
+        // Note that to raise a planet to a metropole, you have to reach the minimum stability
+        // while suffering from the non-metropole penalty below.
         EffectsGroup
-            scope = MaximumNumberOf number = (NamedInteger name = "METROPOLES_PLANET_LIMIT" value = 6) sortkey = LocalCandidate.Population condition = And [
+            scope = MaximumNumberOf number = min((NamedInteger name = "METROPOLES_PLANET_LIMIT" value = 6),
+                                                 floor(Statistic Count condition = And [
+                                                           Planet
+                                                           OwnedBy empire = Source.Owner
+                                                           Species
+                                                       ] / (NamedInteger name = "METROPOLES_MIN_PLANETS" value = 4)))
+                                    sortkey = LocalCandidate.Population condition = And [
                 Planet
                 OwnedBy empire = Source.Owner
                 Population low = (NamedReal name = "METROPOLES_MINIMUM_POPULATION" value = 20.0)
+                Happiness low = (NamedReal name = "METROPOLES_MIN_STABILITY" value = 12)
```

Comment:
I'm a bit concerned about having the selection of which planets from those eligible be just based on population, when also filtering based on population and happiness. Not sure if it will lead to a weird / unexpected situation, or if it will be fine...

Reply:
I found it odd that when you conquer a planet, you get the bonus immediately. Almost all bonuses have a minimum stability, why not this? You have to watch two values instead of one, but Metropoles are important enough to take a little care.

Reply:
I don't have a specific problem, just a vague concern, so it's fine for now...